metron-streaming/Metron-DataLoads/src/test/java/org/apache/metron/dataloads/nonbulk/taxii/flatfile/SimplEnrichmentFlatFileLoaderTest.java

```diff
@@ -0,0 +1,147 @@
+/**
```

Comment:
You are right. It suppose to be in that folder. Mixed up while do the merge yesterday.


Frank Lu
ylu@hortonworks.com
(281) 468-7856

[page2image12144]

From: Casey Stella <notifications@github.com<mailto:notifications@github.com>>
Reply-To: apache/incubator-metron <reply@reply.github.com<mailto:reply@reply.github.com>>
Date: Tuesday, April 19, 2016 at 12:38 PM
To: apache/incubator-metron <incubator-metron@noreply.github.com<mailto:incubator-metron@noreply.github.com>>
Cc: Yi Lu <ylu@hortonworks.com<mailto:ylu@hortonworks.com>>
Subject: Re: [apache/incubator-metron] Dataload testing (#83)


In metron-streaming/Metron-DataLoads/src/test/java/org/apache/metron/dataloads/nonbulk/taxii/flatfile/SimplEnrichmentFlatFileLoaderTest.java<https://github.com/apache/incubator-metron/pull/83#discussion_r60275671>:

> @@ -0,0 +1,147 @@
> +/**


Why is this file moved to org.apache.metron.dataloads.nonbulk.taxii from org.apache.metron.dataloads.nonbulk?

—
You are receiving this because you authored the thread.
Reply to this email directly or view it on GitHub<https://github.com/apache/incubator-metron/pull/83/files/119c70c8cca12e3723c8b3ce9e9c9af3b915b6a0#r60275671>